float64 data
---
bool success
